content/docs/querying/operators.md

```diff
@@ -195,7 +196,7 @@ If the metric `http_requests_total` had time series that fan out by
 `application`, `instance`, and `group` labels, we could calculate the total
 number of seen HTTP requests per application and group over all instances via:
 
-    sum(http_requests_total) by (application, group)
+    sum(http_requests_total) without (instance)
```

Comment:
I'm aware of the benefits of `without` but also still have my doubts about the "always" part.
It just needs one inconsiderate change to a metric's label set to break the semantics of a lot of recording rules.